extensions/eclipse-che-theia-testing-service-ext/src/common/test-protocol.ts

```diff
@@ -0,0 +1,78 @@
+/********************************************************************************
+ * Copyright (C) 2020 RedHat and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import * as vst from 'vscode-languageserver-types';
+
+// Expose additional API that allows you to know if a language server is connected and build a map of the language servers
+export interface LanguageMainTestInterface {
+    // tslint:disable-next-line:no-any
+    $provideCompletionItems(pluginID: string, model: monaco.editor.ITextModel, position: monaco.Position,
```

Comment:
I don't think we should use stuff from the monaco workspace here. This API should be pretty much the same as LanguagesExt, but with the "handle" replaced by "pluginId".